src/actions/deploy-package/index.ts

```diff
@@ -24,7 +24,7 @@ export async function main(): Promise<void> {
             credentials: getCredentials(),
             environmentUrl: getEnvironmentUrl(),
             packagePath: parameterMap["package"],
-        }, runnerParameters, new ActionsHost())
+        }, runnerParameters, new ActionsHost('DeployPackage'))
```

Comment:
non-blocking nit: With the name not containing artifact at first glance the meaning/usage of this string is not clear.
```suggestion
        }, runnerParameters, new ActionsHost('DeployPackageArtifact'))
```

Reply:
it's kinda redundant since the name only shows under the "Artifacts" of the run summary, so no naming conflict